### PR TITLE
PORTALS-3062: add cancer complexity help text content, and also fix t…

### PR DESCRIPTION
**Pull request by jay-hodgson**
…he size of the help popover icon in this context

### CODE REVIEW

File: apps/portals/cancercomplexity/src/config/routesConfig.ts [hunk 1]
```diff
@@ -474,68 +507,7 @@ const routes: GenericRoute[] = [
           {
             path: 'DetailsPage',
             exact: false,
-            synapseConfigArray: [
```
Comment (jay-hodgson) [inline apps/portals/cancercomplexity/src/config/routesConfig.ts:477]: Moved to datasets.ts, and added one helpText.  Attempted to move others, but I ran into cyclical dependencies

File: apps/synapse-portal-framework/src/components/ComponentRenderer.tsx [hunk 2]
```diff
@@ -55,10 +56,9 @@ export function ComponentRenderer(props: { config: SynapseConfig }) {
     <h2 className={`title ${centerTitle ? 'center-title' : ''}`}>
       {title}
       {helpText && (
-        <>
-          {' '}
+        <Box sx={{ fontSize: '14px', display: 'inline-block', ml: '5px' }}>
```
Comment (jay-hodgson) [inline apps/synapse-portal-framework/src/components/ComponentRenderer.tsx:59]: Wrap with a container so we can set the margin and font size

**Comment by jay-hodgson** [comment]
image

**Review by nickgros — APPROVED** [review]
LGTM!

**Pull request merged**